WORKSPACE.in

```diff
@@ -94,6 +94,11 @@ new_http_archive(
     strip_prefix = "opencv-2.4.13.2",
     url = "file:///home/tmp/opencv-2.4.13.2.zip",
 )
+#new_local_repository(
```

Comment:
this change will fail to compile //modules/tools/image_decompress, etc.

Reply:
I didn't found target "//modules/tools/image_decompress" in "v3.1_dev". Then I checked it in "master" and found it. This PR will merge to "v3.1_dev". Finally, changes in WORKSPACE.in are commented out, from bazel's point of view this file is the same as before.